compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -316,7 +316,7 @@ internal class RootNodeOwner(
         override val inputModeManager get() = platformContext.inputModeManager
         override val clipboardManager = PlatformClipboardManager()
         override val accessibilityManager = DefaultAccessibilityManager()
-        override val graphicsContext: GraphicsContext = GraphicsContext()
+        override val graphicsContext = GraphicsContext(this@RootNodeOwner.snapshotObserver.observer)
```

Comment:
It seems we can create own observer, no need to reuse one from `OwnerSnapshotObserver`

Reply:
Let's do this in a separate PR